packages/experimental/Menu/src/MenuPopover/MenuPopover.types.ts

```diff
@@ -1,8 +1,12 @@
 import { IViewProps } from '@fluentui-react-native/adapters';
-import { ICalloutProps } from '@fluentui-react-native/callout';
+import { ICalloutProps, ICalloutTokens } from '@fluentui-react-native/callout';
 
 export const menuPopoverName = 'MenuPopover';
 
+// Support for anchorRect and beakWidth will come at a later time.
+// Omitting dismissBehaviors as it doesn't seem to make sense as a token
+export type MenuPopoverTokens = Omit<ICalloutTokens, 'anchorRect' | 'beakWidth' | 'dismissBehaviors'>;
```

Comment:
Could we also just modify CalloutTokens to not set those 3 as tokens? I suppose that's a breaking change but perhaps nobody has used it that way? 

And for my own understanding: Do we want to eventually make a new native module for Popover, or keep depending on Callout?